code/modules/clothing/spacesuits/rig/modules/ds13/healthbar.dm

```diff
@@ -6,16 +6,14 @@
 	icon_state = "healthbar"
 	use_power_cost = 0
 	origin_tech = list(TECH_MAGNET = 3, TECH_BIO = 3, TECH_ENGINEERING = 5)
-	suit_overlay_inactive = "healthbar_100"
-	suit_overlay_active = "healthbar_100"
-	suit_overlay_used = "healthbar_100"
-	suit_overlay = "healthbar_100"
+	suit_overlay = "healthbar"
+	second_overlay =  "healthbar_100"
```

Comment:
second overlay? whats going on with the overlay system here? have you made changes to it?